packages/react-data-grid/src/Cell.js

```diff
@@ -139,6 +141,22 @@ class Cell extends React.Component {
     }
   };
 
+  onKeyDown = (e) => {
+      // we need to prevent default as we control grid scroll
+    // otherwise it moves every time you left/right which is janky
+    e.preventDefault();
```

Comment:
should only prevent default if we are handling, otherwise we could prevent other expected beahviour to stop working, though in most cases, taht will still be ok through propogation (thinking tab / shift tab, etc)